advocacy_docs/partner_docs/ThalesCipherTrustManager/05-UsingThalesCipherTrustManager.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: 'Using'
+description: 'Walkthrough of example usage scenarios'
+---
+
+After you have configured all of the Thales CipherTrust Manager keys as stated in the Configuring section, you will be able to use them in conjunction with EDB Postgres Advanced Server 15.2.
+
+!!! Note
+    It is important to note that this doc is intended for versions 15.2 of EDB Postgres Advanced Server as this version has the necessary addition of TDE.
+
+Before you can implement Thales CipherTrust Manger with EDB Postgres Advanced Server 15.2 you must ensure that you have the following downloaded to your system:
+- Python
+- [pykmip](https://pypi.org/project/PyKMIP/#files)
+- edb_tde_kmip_client.py
+
+All of the `*.pem` files that you created above need to be copied to your system where your EDB Postgres Advanced Server 15.2 instance is installed.  For this guide, all of the `*.pem` files are in the `/tmp/` directory as well as the `edb_tde_kmip_client.py` file.
+
+## Create pykmip.conf File
+
+1. On your system where you have your EDB Postgres Advanced Server 15.2 instance, navigate to the directory where you have saved your `*.pem` files and the `edb_tde_kmip_client.py` client.
+
+2. In that directory create a file called `pykmip.conf` and input the following:
+- Host
+- Port
+- Username
+- Password
+- Keyfile
+- Certfile
+- Ca_certs
+
+For example:
```

Comment:
the example looks funny on my screen.  Over sized characters.  not the black terminal look that hashicorp has.